internal/bminventory/inventory_test.go

```diff
@@ -5629,7 +5629,7 @@ func (e eventMsgMatcher) String() string {
 	return "input contains substrings"
 }
 
-var _ = Describe("AMS subscriptions", func() {
+var _ = FDescribe("AMS subscriptions", func() {
```

Comment:
Don't commit FDescribe and FIt usage

Reply:
Thanks, it is making the CI fail anyway so I don't put too much attention to it :)

Reply:
Fixed.